invenio_rdm_records/services/components/metadata.py

```diff
@@ -19,6 +20,9 @@ class MetadataComponent(ServiceComponent):
 
     field = "metadata"
     new_version_skip_fields = ["publication_date", "version"]
+    new_version_generated_fields = {
```

Comment:
minor: why not removing `publication_date` from `new_version_skip_fields`? It would simply import the previous version's publication date and then the user could modify it.

Reply:
I considered that initially, but this is how we had it in legacy and is the reason we ended up with many records that just had the exact same `publication_date` across all versions.